osbenchmark/worker_coordinator/runner.py

```diff
@@ -1848,32 +1849,43 @@ def __repr__(self, *args, **kwargs):
         return "delete-async-search"
 
 
-class OpenPointInTime(Runner):
+class CreatePointInTime(Runner):
     async def __call__(self, opensearch, params):
         op_name = mandatory(params, "name", self)
         index = mandatory(params, "index", self)
         keep_alive = params.get("keep-alive", "1m")
-        response = await opensearch.open_point_in_time(index=index,
-                                         params=params.get("request-params"),
-                                         keep_alive=keep_alive)
+        response = await opensearch.create_point_in_time(index=index,
+                                                         params=params.get("request-params"),
+                                                         keep_alive=keep_alive)
         id = response.get("id")
         CompositeContext.put(op_name, id)
 
     def __repr__(self, *args, **kwargs):
-        return "open-point-in-time"
+        return "create-point-in-time"
 
 
-class ClosePointInTime(Runner):
+class DeletePointInTime(Runner):
     async def __call__(self, opensearch, params):
         pit_op = mandatory(params, "with-point-in-time-from", self)
         pit_id = CompositeContext.get(pit_op)
         request_params = params.get("request-params", {})
-        body = {"id": pit_id}
-        await opensearch.close_point_in_time(body=body, params=request_params, headers=None)
+        body = {
+            "pit_id": [pit_id]
+        }
+        await opensearch.delete_point_in_time(body=body, params=request_params, headers=None)
         CompositeContext.remove(pit_op)
 
     def __repr__(self, *args, **kwargs):
-        return "close-point-in-time"
+        return "delete-point-in-time"
+
+
+class GetPointInTime(Runner):
```

Comment:
Should we make it list all? This will be in-sync with the changes being made in opensearch backend

Reply:
Done,thanks!